doc/api/child_process.md

```diff
@@ -36,8 +36,10 @@ identical to the behavior of pipes in the shell. Use the `{ stdio: 'ignore' }`
 option if the output will not be consumed.
 
 The command lookup is performed using the `options.env.PATH` environment
-variable if it is in the `options` object. Otherwise, `process.env.PATH` is
-used.
+variable if `env` is in the `options` object. Otherwise, `process.env.PATH` is
+used. If `options.env` is set, but without `PATH`, lookup on UNIX is performed
+on a default search path search of `/usr/bin:/bin` (see execvpe/execvp), on Windows
```

Comment:
> (see execvpe/execvp)

is there a link you can include here?

Reply:
There are a bunch of "mirrors" on the net, but I'm not sure what the official would be (and maybe not contain ads):
- https://man7.org/linux/man-pages/man3/execvpe.3.html
- https://linux.die.net/man/3/execvpe
- https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/execvp.3.html

Maybe it could just say to consult your operating system manual?

Reply:
Updated proposed docs with comment about execvp